xdsl/dialects/llvm.py

```diff
@@ -92,6 +92,29 @@ def typed(type: Attribute):
         return LLVMPointerType([type, NoneAttr()])
 
 
+
+
+@irdl_op_definition
+class GetElementPtrOp(Operation):
+    name = "llvm.getelementptr"
+    
+
+    ptr: Annotated[Operand, LLVMPointerType]
+
+
+    def get(cls,
+            ptr: Operation,
```

Comment:
We usually do Operation and SSAValue. If you just want one, please use SSAValue instead of Operation.